File: app/src/main/java/me/varlez/movies/common/Logger.java
package me.varlez.movies.common;

import android.util.Log;

/**
 * Utility class for simple logging features.
 */
public final class Logger {

    private static final String LOG_TAG = "Movies";

    private Logger() {
    }

    public static void d(String message, Object... args) {
        Log.d(LOG_TAG, String.format(message, args));
    }

    public static void e(String message, Object... args) {
        Log.e(LOG_TAG, String.format(message, args));
    }
}

File: app/src/main/java/me/varlez/movies/common/Logger.kt
package me.varlez.movies.common

import android.util.Log

/**
 * Utility class for simple logging features.
 */
object Logger {

    private val LOG_TAG = "Movies"

    fun d(message: String, vararg args: Any) {
        Log.d(LOG_TAG, String.format(message, *args))
    }

    fun e(message: String, vararg args: Any) {
        Log.e(LOG_TAG, String.format(message, *args))
    }
}
